mambaSharedFramework/ValueTypes.swift

```diff
@@ -240,29 +240,29 @@ public struct HLSCodecArray: Equatable, FailableStringLiteralConvertible {
     }
 
     public func containsAudio() -> Bool {
-        return HLSCodec.audioPrefixes.filter( { audioPrefix in return self.codecs.filter( { $0.codecDescriptor.hasPrefix(audioPrefix) } ).count > 0 } ).count > 0
+        return CodecValueType.audioPrefixes.filter( { audioPrefix in return self.codecs.filter( { $0.codecDescriptor.hasPrefix(audioPrefix) } ).count > 0 } ).count > 0
     }
    
     public func containsVideo() -> Bool {
-        return HLSCodec.videoPrefixes.filter( { videoPrefix in return self.codecs.filter( { $0.codecDescriptor.hasPrefix(videoPrefix) } ).count > 0 } ).count > 0
+        return CodecValueType.videoPrefixes.filter( { videoPrefix in return self.codecs.filter( { $0.codecDescriptor.hasPrefix(videoPrefix) } ).count > 0 } ).count > 0
     }
     
     public func containsAudioVideo() -> Bool {
         return self.containsAudio() && self.containsVideo()
     }
     
-    public init(codecs: [HLSCodec]) {
+    public init(codecs: [CodecValueType]) {
         self.codecs = codecs
     }
 }
 
-extension HLSCodecArray {
-    public func includes(codec: HLSCodec) -> Bool {
+extension CodecValueTypeArray {
+    public func includes(codec: CodecValueType) -> Bool {
         return codecs.contains(codec)
     }
 }
 
-public func ==(lhs: HLSCodecArray, rhs: HLSCodecArray) -> Bool {
+public func ==(lhs: CodecValueTypeArray, rhs: CodecValueTypeArray) -> Bool {
     return lhs.codecs == rhs.codecs
 }
```

Comment:
Entire file, naming inconsistency WRT not replacing `HLS` with `Playlist`

Reply:
Everything in the file is not required to be part of a playlist, so I think the naming is good.